internal/version/version.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright © 2022 - 2025 SUSE LLC
+SPDX-License-Identifier: Apache-2.0
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package version
+
+import (
+	"runtime"
+)
+
+var (
+	version = "v0.0.1"
+	// gitCommit is the git sha1
+	gitCommit = ""
+)
+
+// BuildInfo describes the compile time information.
+type BuildInfo struct {
```

Comment:
Out of curiosity, what do you think about reading this data from the `runtime/debug` package directly?

Reply:
I did not find a way to read the tag from `runtime/debug`

Reply:
This is usually the case because you can have multiple tags on the same commit. I'm good with the simplified version, it's nice!